keyboards/kinesis/keymaps/TuesdayJohn/keymap.c

```diff
@@ -0,0 +1,509 @@
+  _______, COLEMAK, QWERTY,  _______, COLEMKG, QWERTYG,                                                       NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______, _______,                                                       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                                                       _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                                                       _______, _______, _______, _______, _______, _______,
+           _______, _______, _______, _______,                                                                         _______, _______, _______, _______, 
+                                                        _______, _______,                   _______, _______,
+                                                                 _______,                   _______,
+                                               _______, _______, _______,                   _______, _______, _______
+),
+
+[_ADJUST2] = LAYOUT_pretty(
+  _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+  _______, COLEMAK, QWERTY,  _______, COLEMKG, QWERTYG,                                                       NUMPAD,  _______, _______, _______, _______, RESET,
+  _______, _______, _______, _______, _______, _______,                                                       _______, _______, _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                                                       _______, NKROTG,  _______, _______, _______, _______,
+  _______, _______, _______, _______, _______, _______,                                                       _______, _______, _______, _______, _______, _______,
+           _______, _______, _______, _______,                                                                         _______, _______, _______, _______, 
+                                                        _______, _______,                   _______, _______,
+                                                                 _______,                   _______,
+                                               _______, _______, _______,                   _______, _______, _______
+)
+
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _NUMBERS, _NUMBERS2, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+	case COLEMAK:
+      if (record->event.pressed) {
+        default_layer_set(1UL << _COLEMAK);
+//        persistent_default_layer_set(1UL << _COLEMAK);
+        layer_off ( _QWERTY);
```

Comment:
You could just use `layer_move(0)`, I think.

Reply:
So just using ‘layer